frontend/src/tasks/LoadRestrictionsDatesTask.js

```diff
@@ -0,0 +1,23 @@
+import axios from "axios";
+
+class LoadRestrictionsDatesTask {
+  #getRestrictionDates = async (iso_code) => {
+    try {
+      const res = await axios.get(
+        `http://localhost:8000/api/modeldaterange/?iso_code=${iso_code}`,
+        {}
+      );
+      return res.data;
+    } catch (error) {
+      console.log(error);
+      return [];
+    }
+  };
+
+  getCountryRestrictionDates = async (iso_code) => {
+    const data = await this.#getRestrictionDates(iso_code);
+    return data;
```

Comment:
Can this just be `return await this.#getRestrictionDates(iso_code);`? I'm not sure whether that would work or not.

Reply:
will test this in PR #77